src/main/java/seedu/address/logic/commands/LoadCourtCommand.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.image.Image;
+
+/**
+ * Loads an image as a background image for use in StrategyPanel.
+ */
+public class LoadCourtCommand extends Command {
+    public static final String COMMAND_WORD = "load-court";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Load an image from '/courts/' directory and sets as background image for the Strategy Panel. "
+            + "Parameters: "
+            + "IMAGE_NAME (" + Image.MESSAGE_CONSTRAINTS + ") "
+            + "Example: " + COMMAND_WORD + " "
+            + "basketball";
+
+    public static final String MESSAGE_SUCCESS = "Image has been loaded: %1$s";
+
+    private final Image image;
+
+    /**
+     * Constructor for LoadCourtCommand class.
+     * @param image The image file.
+     */
+    public LoadCourtCommand(Image image) {
+        requireAllNonNull(image);
+
+        this.image = image;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireAllNonNull(model);
+        model.replaceBackgroundImage(image);
```

Comment:
I think line 39 can be removed if you find my suggestion below good and would like to utilise it. Likewise for all the methods in logic and model relating to background image